src/style.rs

```diff
@@ -41,7 +41,7 @@
 //!         // Red background
 //!         SetBackgroundColor(Color::Red),
 //!         // output text
-//!         Output("Styled text here.".to_string()),
+//!         Print("Styled text here.".to_string()),
```

Comment:
Isn't "Styled text here" a bit misleading here? Sounds like `PrintStyledContent`. Maybe "Blue on red" will be better?